bindings/java/gateway-java-binding/src/main/java/com/microsoft/azure/gateway/core/Broker.java

```diff
@@ -10,46 +10,38 @@
 
 public class Broker {
 
-    //Loads the native library
-    static {
-        System.loadLibrary("java_module_host");
-    }
+    private long brokerAddr;
+    private LocalBroker localBroker;
```

Comment:
ProxyBroker used by remote module extends Broker and this is to avoid loading java_module_host.dll which is not used by remote module.